website/docs/software/setup-guide/soft-setup-guide_draft.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Getting started 
+sidebar_position: 1
+---
+
+# Software setup guide
+
+## Supported Environment
+
+### SDK Requirements
+- **openarm_sdk**: Works on Ubuntu with SocketCAN interface devices
+  - Tested on Ubuntu 22.04 
+  - **TODO**: Test compatibility on Ubuntu 24.04 (@kevin)
+- **CMake 3.22+** required for building
+
+### Other Package Dependencies
+- **openarm_ros2**: Compatible with ROS2 Humble
+- **openarm_isaac** (Work in Progress): Requires Ubuntu 22.04 + ROS2 Humble-based bridge
+
+## Building the SDK
+
+### 1. Clone and Initialize Repository
+```bash
+git clone https://github.com/enactic/openarm_sdk.git
+cd openarm_sdk
+git submodule update --init --recursive
+```
+
+### 2. Build the CAN Library
+```bash
+cd openarm_can
+mkdir build
+cd build
+cmake ..
+make
+sudo make install
+```
+
+### 3. Optional: ROS2 Integration
+- If using URDF files directly, no additional configuration needed
+- For custom ROS2 setup with xacro, DAE files, etc., configure ROS2 workspace accordingly
+
+## Pre-Flight Configuration
+
+### Motor ID Configuration
+**CRITICAL**: Configure Damiao motor IDs before running any code on the arm.
```

Comment:
We can use the admonitions syntaxes here:

https://docusaurus.io/docs/markdown-features/admonitions

```suggestion
:::warning
Configure Damiao motor IDs before running any code on the arm.
:::
```